app/blueprints/fund/forms.py

```diff
@@ -37,16 +38,62 @@ def __init__(self, name: str, _prefix, _translations, label: str, _form, source_
 
 class FundForm(FlaskForm):
     fund_id = HiddenField("Fund ID")
-    name_en = StringField("Grant name", validators=[DataRequired()])
-    name_cy = StringField("Grant name (Welsh)")
-    title_en = StringField("Application heading", validators=[DataRequired()])
-    title_cy = StringField("Application heading (Welsh)")
+    welsh_available = RadioField(
+        "Is this grant available in Welsh?",
+        choices=[("true", "Yes"), ("false", "No")],
+        widget=GovRadioInput(),
+        default="false",
+        render_kw={"class": "govuk-radios govuk-radios--inline"},
+    )
+    name_en = StringField(
+        "Grant name",
+        widget=GovTextInput(),
+        description="For example, Community Ownership Fund",
+        validators=[DataRequired()],
+    )
+    name_cy = StringField(
+        "Grant name (Welsh)", widget=GovTextInput(), description="Er enghraifft, Gronfa Perchnogaeth Gymunedol"
```

Comment:
Let's keep instructions in English, and example content in Welsh please 🙏 

```suggestion
        "Grant name (Welsh)", widget=GovTextInput(), description="For example, Gronfa Perchnogaeth Gymunedol"
```